src/libbson/src/bson/bson-error.c

```diff
@@ -116,6 +116,10 @@ bson_strerror_r (int err_code,                    /* IN */
    if (strerror_s (buf, buflen, err_code) != 0) {
       ret = buf;
    }
+#elif defined(_AIX)
+   // AIX does not provide strerror_l, and its strerror_r isn't glibc's.
+   // But it does provide a glibc compatible one called __linux_strerror_r
+   ret = __linux_strerror_r (err_code, buf, buflen);
```

Comment:
@NattyNarwhal: did you guys only come across this after upgrading to libmongoc 1.24? It looks like it may be related to the recent changes in [CDRIVER-4679](https://jira.mongodb.org/browse/CDRIVER-4679), which were prompted by [PHPC-2263](https://jira.mongodb.org/browse/PHPC-2263).

Reply:
I finally had the opportunity to upgrade from whatever the vendored version in 1.12 of the PHP driver was to whatever the version in 1.16 was, so I'd be running in context of that release of the PHP driver's configure script.

Reply:
Thanks. If you jumped from 1.12 to 1.16 I think it's safe to assume this issue was introduced by the aforementioned fixes for Alpine Linux. :v: